src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js

```diff
@@ -62,7 +62,7 @@ class ExpandableRulesCard extends React.Component {
                         <SplitItem> <Battery label='Impact' severity={ rule.impact.impact } /> </SplitItem>
                         <SplitItem> <Battery label='Likelihood' severity={ rule.likelihood } /> </SplitItem>
                         <SplitItem> <Battery label='Total Risk' severity={ rule.severity } /> </SplitItem>
-                        <SplitItem><Battery label='Risk Of Change' severity={ rule.resolution_risk } /></SplitItem>
+                        <SplitItem> <Battery label='Risk Of Change' severity={ report.resolution.resolution_risk.risk } /> </SplitItem>
```

Comment:
Can we use `<Section type="icon-group__with-major">` from the `<Section>` Component in FEC for all of these Batteries instead of SplitItem?

Reply:
Disregard original ss @ryelo this ok with you? 
<img width="1288" alt="screen shot 2019-01-16 at 12 41 23 pm" src="https://user-images.githubusercontent.com/6640236/51267734-6e68f980-198c-11e9-9be3-f42b6326a472.png">

Reply:
Yep! Awesome!